packages/KTable/KTable.vue

```diff
@@ -164,6 +171,20 @@ export default {
     sortKey: {
       type: String,
       default: ''
+    },
+    /**
+     * A function that conditionally specifies row attributes on each row
+     */
+    rowAttrs: {
+      type: Function,
+      default: () => ({})
+    },
+    /**
+     * A prop that enables a side border with a themable color to it.
+     */
+    hasSideBorder: {
```

Comment:
I think @darrenjennings was thinking this should be true by default. I agree if all tables have it